package.json

```diff
@@ -48,7 +49,7 @@
   "scripts": {
     "build": "tsc",
     "format": "prettier --write .",
-    "test": "ava --timeout=2m build/**/*.test.js",
+    "test": "npm run build && ava --timeout=2m build/**/*.test.js",
```

Comment:
please drop this

Reply:
Sure - though basing tests on the compiled code can be quite frustrating if the test suite isn't building it as well. I'll revert, but this made it a lot easier to write the tests.